cmd/run/message_input.go

```diff
@@ -123,6 +141,15 @@ func (mi *MessageInput) launchEditorAction() {
 			return
 		}
 	})
+	
+	// One may ask "Why don't we remove the file earlier?". Well,
+	// the file won't contain any message up until this point, and the file
+	// is created in /tmp anyway so it'll be cleared on a reboot.
+	var msg, read_err = os.ReadFile(f.Name())
```

Comment:
Use `:=` instead of `var`.

```suggestion
	msg, read_err := os.ReadFile(f.Name())
```